### PR TITLE
Add: "Deploying Llama 3.2 3B in a Kubernetes (K8s) cluster"

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
## Description

This PR adds a tutorial: "Deploying Llama 3.2 3B in a Kubernetes (K8s) cluster"

## Related issues

None

## Checklist

- [x] Documentation follows the [contributing guidelines](../CONTRIBUTING.md).
- [x] Documentation linked to in nav and index pages.
- [x] Files used in this PR (e.g., images and PDFs) are committed.
- [x] Link to existing documentation and the reverse.
- [x] Changes have been tested locally.
- [x] Reviewers have been assigned (at least 1 tech writer and 1 eng).

## Additional notes

None

### CODE REVIEW

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+# Deploying Llama 3.2 3B in a Kubernetes (K8s) cluster
+
+## Introduction
+
+In this tutorial, you'll:
+
+1. Stand up a single-node Kubernetes cluster on an
+   [on-demand instance :octicons-link-external-16:](https://lambdalabs.com/service/gpu-cloud){target="_blank"}
+   using [K3s :octicons-link-external-16:](https://k3s.io/).
```
Comment (ta11t0m) [inline docs/education/large-language-models/k8s-ollama-llama-3-2.md:18]: Suggest defining K3s: 
"K3s is a lightweight, certified Kubernetes distribution that is designed to run Kubernetes clusters with minimal resource overhead."

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+description: Learn how to deploy Llama 3.2 3B in a Kubernetes cluster.
+tags:
+  - api
+  - kubernetes
+  - llama
+  - llm
+---
+
+# Deploying Llama 3.2 3B in a Kubernetes (K8s) cluster
+
```
Comment (ta11t0m) [inline docs/education/large-language-models/k8s-ollama-llama-3-2.md:11]: Consider adding an introductory paragraph describing why you would want to run llama3 in a k8s cluster, maybe mention things like: 
- k3s helps if you're in a resource-constrained environment with lower mem/cpu requirements
- lightweight, easy to install/maitain
- k3s horizontally scales pods to meet traffic demand

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+    ```
+
+    You should see output similar to:
+
+    ```{.text .no-copy}
+    NAME              STATUS   ROLES                  AGE    VERSION
+    104-171-203-164   Ready    control-plane,master   100s   v1.30.5+k3s1
+    ```
+
+1. Install socat by running:
```
Comment (ta11t0m) [inline docs/education/large-language-models/k8s-ollama-llama-3-2.md:60]: style question/suggestion- should we use code font for`socat`, like they do here: https://www.redhat.com/sysadmin/getting-started-socat

**Review by ta11t0m — COMMENTED** [review]
Nice! I left a few questions and suggestions.

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md (hunk 3 above)
Comment (cbrownstein-lambda) [reply docs/education/large-language-models/k8s-ollama-llama-3-2.md:60]: I struggle with this: sometimes I feel it makes sense to style the names of command-line programs and packages as `code`. Sometimes I feel the names of programs and packages—both GUI programs and command-line programs—should be unstyled.

Let's style as `code` and remain consistent.

```suggestion
1. Install `socat` by running:
```

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md (hunk 1 above)
Comment (ta11t0m) [reply docs/education/large-language-models/k8s-ollama-llama-3-2.md:18]: Your call, Cody.

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+1. Deploy
+   [Ollama :octicons-link-external-16:](https://ollama.com/){target="_blank"} in
+   your cluster to serve the
+   [Llama 3.2 3B model :octicons-link-external-16:](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/){target="_blank"}.
+1. Install the Ollama client.
+1. Interact with the Llama 3.2 3B model.
+
+!!! note
+
+    You don't need a Kubernetes cluster to run Ollama and serve the Llama 3.2 3B
```
[the remark this answers is not in the document]
Comment (ta11t0m) [reply docs/education/large-language-models/k8s-ollama-llama-3-2.md:31]: Wondering if you can connect the value of K8s to the Ollama use cases so it's clear why you're discussing the two together.
If k8s isn't useful for Ollama, then maybe the topic of k8s for ODC could be a separate tutorial- having it separate would be good for SEO and repeat uses independent of Ollama.
Non-blocking food for thought.

**Review by ta11t0m — APPROVED** [review]

File: docs/education/large-language-models/k8s-ollama-llama-3-2.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cbrownstein-lambda) [reply docs/education/large-language-models/k8s-ollama-llama-3-2.md:31]: > If k8s isn't useful for Ollama, then maybe the topic of k8s for ODC could be a separate tutorial- having it separate would be good for SEO and repeat uses independent of Ollama.

100%. Work in progress. :smiley:

**Pull request merged**